src/Module/AllFriends.php

```diff
@@ -63,25 +63,22 @@ public static function content(array $parameters = [])
 		DI::page()['aside'] = "";
 		Model\Profile::load($app, "", Model\Contact::getByURL($contact["url"], false));
 
-		$total = Model\GContact::countAllFriends(local_user(), $cid);
```

Comment:
Please keep using the `GContact` methods or include the extra condition they feature in this module.

Reply:
Keeping the `GContact` class would be contra productive :-)

Reply:
That's why I added an alternative!